web/rest-crud-demo/README.md

```diff
@@ -4,13 +4,16 @@ FIXME: description
 
 ## Installation
 
-Download from http://example.com/FIXME.
+- Install (Leiningen)[https://leiningen.org/] as a package manager.
+- etc (TBD)
 
 ## Usage
 
-FIXME: explanation
+Running in local
 
-    $ java -jar rest-crud-demo-0.1.0-standalone.jar [args]
+```
+lein start
```

Comment:
lein start didn't work for me. lein run works. am I missing something?

Reply:
also, I'm able to run locally using lein ring server-headless. any thoughts on what's the best approach?

Reply:
TBH, I am not sure which is better.

Reply:
Is there any chance to not use lein? In the cli subproject we tried to stick to deps.edn, and I'm curious to see if and how we can implement hot-reloading with it...